src/utils/convert.js

```diff
@@ -1,17 +1,9 @@
-const { Block, Inline, Leaf, Mark, Text } = require('slate');
+const { Block, Document, Inline, Leaf, Mark, Text, Value } = require('slate');
 const isPlainObject = require('is-plain-object');
```

Comment:
I think this can be removed as well? (Given that `isElement` is no longer needed ...)

Reply:
Good point, removing it.